JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/api/HttpServer.java

```diff
@@ -529,6 +558,7 @@ private Route createRoutes(final ActorSystem<Void> actorSystem, final ActorRef<B
                                                         () -> routeMatchesForReviewList(actorSystem, backEnd)),
                                                 path("Document",
                                                         () -> routeDocument(actorSystem, backEnd)),
+                                                path("Search", () -> routeSearch(actorSystem, backEnd)),
```

Comment:
```suggestion
                                                path("patient", () -> routeSearch(actorSystem, backEnd)),
```

Reply:
@mahaomolise92 I also noticed that you created a get endpoint for search. Should this not be a post?